app/schemas.py

```diff
@@ -39,6 +39,7 @@ class QuizLanguage(Enum):
 class QuizType(Enum):
     assessment = "assessment"
     homework = "homework"
+    omr = "omr-assessment"
```

Comment:
This is a bit weird. There is a lot of behaviour difference between `assessment` and `homework`, but OMR will now be supported only for one sort of behaviour (`assessment`). 

Is that what we want?

Reply:
Yes. OMR flow makes sense only for assessments.
For homeworks, once a question is answered, we cannot change answer again. I wonder if that would look good for a "omr-homework" quiz type. What if user accidentally touches a bubble, etc.

Also homeworks are for at-home learning. We can assume students will have enough time to go through questions.

We also send fewer questions in our homeworks currently (like 5-10). It should be easy to navigate across them quickly in question player view.